src/lib/colorUtils.ts

```diff
@@ -0,0 +1,11 @@
+import * as _ from 'lodash'
+import { SiteVariablesInput, ColorVariants, ColorValues } from '../themes/types'
+
+export const mapColorsToScheme = <T>(
+  siteVars: SiteVariablesInput,
+  mapper: string | number | keyof ColorVariants | ((color: ColorVariants) => T),
```

Comment:
It was wrong advice from my side to use `string`, in fact the valid signature is below:

```suggestion
  mapper: keyof ColorVariants | ((color: ColorVariants) => T),
```

Reply:
done